### ISSUE TITLE
Open .km file #bug

### ISSUE THREAD

**Issue by Dubidubiduu** [ACTom/files_mindmap#21]
I'm using https://apps.nextcloud.com/apps/files_mindmap 0.0.11 with NC 14.0.3 and I wasn't able to open any .km file again after creating it.. :-/

When I opened that file with text editor for me it seems quite good, I just wasn't able to open it with the app again.

**Comment by ACTom** [comment]
When install this app, it need to add something in config/mimetypealiases.json and config/mimetypemapping.json, do you have permission to modify config directory?

**Comment by Dubidubiduu** [comment]
Set to 0750

**Comment by cobiam** [comment]
Good day, first congratulations for the mindmaps app for Nextcloud, i am trying to open a file with format xmind through your app. "it need to add something in config/mimetypealiases.json and config/mimetypemapping.json", what codes lines i must write into these files of config?
Thank you for your collaboration and support.

**Comment by DerekCaelin** [comment]
![image](https://user-images.githubusercontent.com/10799815/55675474-06da9000-5891-11e9-847d-44c0dc8308f1.png)

Same issue. I can create new mindmaps, and I can edit old mindmaps, but now once a mindmap is edited, NextCloud does doesn't view it as a viewable/editable file.

App version: 0.0.1.1
Nextcloudpii version: 1.11.1
Nextcloud version: 15.0.6.1

**Comment by DerekCaelin** [comment]
Has this issue been resolved?

**Comment by ACTom** [comment]
@DerekCaelin yes, fix it in v0.0.12

**Comment by DerekCaelin** [comment]
Thank you!

**Comment by harryscollar** [comment]
A few days ago I updated to nextcloud 15.0.5 to 15.0.6 and my mindmaps stopped working.

The icons had reverted back to generic document icons and clicking on them did a download action instead of opening them as a mind map.

The next day, I then updated to 15.0.7 and updated MindMap to 0.0.13.

My older mindmaps still don't work but newer ones do.

I've also noticed that if you edit a .km file with a text editor it then also becomes a generic document.

Is there some way to manually re-associate a .km file as a mindmap file or somehow salvage old mindmap files?
